### PR TITLE
Nvcustomname_backend

### PR DESCRIPTION
**Pull request by norvarju**

### CODE REVIEW

**Review by bekobarna — COMMENTED** [review]
Because you no longer use the Faker library,  you can delete the dependency and the imported library line in the KalendarService class

File: build.gradle [hunk 1]
```diff
@@ -33,7 +33,6 @@ dependencies {
 	compile('com.google.oauth-client:google-oauth-client-jetty:1.23.0')
 	compile('com.google.api-client:google-api-client:1.23.0')
 	compile('org.flywaydb:flyway-core:5.0.5')
-	testCompile group: 'com.github.javafaker', name: 'javafaker', version: '0.12'
```
Comment (arnikaeva) [inline build.gradle]: acceptance criteria: if no name is provided a default name will be used (existing solution)

File: src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/service/KalendarService.java [hunk 2]
```diff
@@ -36,8 +35,7 @@
     }
 
     public void setKalendar(Kalendar kalendar, KalendarFromAndroid kalendarFromAndroid, String clientToken) {
-        Faker faker = new Faker();
-        kalendar.setName(faker.gameOfThrones().character());
+        kalendar.setName(kalendarFromAndroid.getCustomName());
```
Comment (arnikaeva) [inline src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/service/KalendarService.java]: if no name is provided, a fake name should be used

File: build.gradle (hunk 1 above)
Comment (arnikaeva) [reply build.gradle]: unit tests are missing, check [Jira](http://jira.greenfox.academy/browse/OPAL-205) definition of done for details.

**Review by arnikaeva — CHANGES_REQUESTED** [review]
Unit tests are missing, custom name should be optional.

**Pull request closed without merging**